src/Plivo/Resource/Message/Message.cs

```diff
@@ -105,6 +105,25 @@ public class Message : Resource
         /// <value>The destination_country_iso2.</value>
         public string DestinationCountryIso2 { get; set; }
 
+        /// <summary>
+        /// Gets or sets the destination_network.
+        /// </summary>
+        /// <value>The destination_network.</value>
+        public string DestinationNetwork { get; set; }
+
+        /// <summary>
+        /// Gets or sets the carrier_fees_rate.
+        /// </summary>
+        /// <value>The carrier_fees_rate.</value>
+        public string CarrierFeesRate { get; set; }
+
+
+        /// <summary>
+        /// Gets or sets the carrier_fees.
+        /// </summary>
+        /// <value>The carrier_fees.</value>
+        public string CarrierFees { get; set; }
```

Comment:
These fields should be added in: **public override string ToString()**
as well...

"CarrierFees: " + CarrierFees + "\n" +
"CarrierFeesRate: " + CarrierFeesRate + "\n" +
"DestinationNetwork: " + DestinationNetwork + "\n";

Reply:
@renoldthomas-plivo done